File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        
        String answer = "";
        
        int idx = 0;
        
        for(int i=0; i<s.length(); i++){
            
            if(idx % 2 == 0){
                answer = answer + Character.toUpperCase(s.charAt(i));
            }else{
                answer = answer + Character.toLowerCase(s.charAt(i));
            }
            idx++;
            if(s.charAt(i) == ' ') idx = 0;
        }
            
            
        return answer;
    }
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    
    var answer = '';
    var idx = 0;
    
    for(let i=0; i<s.length; i++){
        if(idx % 2 == 0){
            answer = answer+ s.charAt(i).toUpperCase();
        }else{
            answer = answer + s.charAt(i).toLowerCase();
        }
        idx++;
        if(s.charAt(i) == ' ') idx = 0;
    }
    
    return answer;
}